server/routes/makeAReferral/form/referralFormPresenter.ts

```diff
@@ -321,15 +326,19 @@ class SectionValues {
   }
 }
 class TaskValues {
-  constructor(private referral: DraftReferral) {}
+  constructor(
+    private referral: DraftReferral,
+    private draftOasysRiskInfo: DraftOasysRiskInformation | null | undefined
+  ) {}
 
   // TODO: IC-1676. We need a field to confirm that the user has "checked" service user details.
   get serviceUserDetails(): DraftReferralValues {
     return []
   }
 
+  // TODO: remove this.referral.additionalRiskInformation once switched over to full risk information
```

Comment:
does 'additionalRiskInformation' dissapear completely when this is rolled out?

Reply:
yes, we can remove it completely once risk is fully rolled out